warehouse/upload.go

```diff
@@ -105,21 +105,33 @@ type Upload struct {
 
 type tableNameT string
 
-type UploadJobT struct {
-	upload               *Upload
+type UploadJobFactory struct {
```

Comment:
Didn't get where are we initialing this.

Reply:
We weren't I was waiting for the destinationHistory client before moving forward with wiring this up.

it is ready now